### PR TITLE
Laurelin potential palindrome

### PR DESCRIPTION
**Pull request by Laurelin**
The markdown was pretty Java-specific, so I'm really unsure about whether I did this correctly here. test is separate from the laurelin.py because that's how pytest works, afaik. I built this with python 3.8.5.

### CODE REVIEW

File: challenges/2021-04-20-potential-palindrome/solutions/laurelin.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+def can_make_palindrome(s: str) -> bool:
+    #count char frequency in string, if more than one is odd, not a palindrome
+    if not s:
+        return False
+    freq = defaultdict(int)
+    for char in s:
+        freq[char] += 1
+    if len([val for val, chr in freq.items() if chr % 2 != 0]) > 1:
```
Comment (parhaml) [inline challenges/2021-04-20-potential-palindrome/solutions/laurelin.py:11]: nice oneliner

**Review by ericwburden — CHANGES_REQUESTED** [review]
This looks good! I like having the test file in here. (I've seen setups in Python with the tests and code in the same file, but it may not have been using pytest). The only change I'm requesting is that you remove the *.pyc (bytecode) files from the branch.

File: challenges/2021-04-20-potential-palindrome/solutions/laurelin.py (hunk 1 above)
Comment (drkennetz) [reply challenges/2021-04-20-potential-palindrome/solutions/laurelin.py:11]: This is nice!

**Review by drkennetz — APPROVED** [review]

**Comment by Laurelin** [comment]
Thanks! I think I've removed the bytecode parts.

**Review by ericwburden — APPROVED** [review]
Perfect!

**Review by parhaml — APPROVED** [review]

**Comment by parhaml** [comment]
@Laurelin if you're ready. Will you remove `wip` from the PR title? Or let me know and I'll merge your solution. Great work! 💯

**Comment by drkennetz** [comment]
@parhaml I took care of it!

**Pull request merged**